apps/communications/contacts/services/contacts.js

```diff
@@ -162,6 +162,17 @@
         }
       };
       cursor.onerror = onErrorCB;
+    },
+
+    getCount: function(callback) {
```

Comment:
I think we can't rid off this function because the `Find duplicates button` should only be enabled when the user has more than one contact in the agenda but the method `isEmpty` will return false when having only one contact so the button will be enabled.